api/envoy/annotations/deprecation.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package envoy.annotations;
+
+import "google/protobuf/descriptor.proto";
+
+// Magic number in this file derived from top 28bit of SHA256 digest of "envoy.annotation.do_not_use"
+extend google.protobuf.FieldOptions {
+  bool do_not_use = 169129351;
+}
+
+// Magic number in this file derived from top 28bit of SHA256 digest of "envoy.annotation.do_not_use_enum"
+extend google.protobuf.EnumValueOptions {
+  bool do_not_use_enum = 162744140;
```

Comment:
I think you can also call this `do_not_use`

Reply:
I don't think I can.
Things defined in the extend field don't get namespace prefixed.  If you notice in the utility file, they're 
envoy::annotations::disallowed_by_default_enum
and envoy::annotations::disallowed_by_default
and they conflict if they are both disallowed_by_default